crates/ruff_linter/src/rules/pyupgrade/rules/outdated_version_block.rs

```diff
@@ -152,41 +161,46 @@ pub(crate) fn outdated_version_block(checker: &mut Checker, stmt_if: &StmtIf) {
                 value: ast::Number::Int(int),
                 ..
             }) => {
-                if op == &CmpOp::Eq {
-                    match int.as_u8() {
-                        Some(2) => {
-                            let mut diagnostic = Diagnostic::new(
-                                OutdatedVersionBlock {
-                                    reason: Reason::Outdated,
-                                },
-                                branch.test.range(),
-                            );
-                            if let Some(fix) = fix_always_false_branch(checker, stmt_if, &branch) {
-                                diagnostic.set_fix(fix);
-                            }
-                            checker.diagnostics.push(diagnostic);
+                let reason = match (int.as_u8(), op) {
+                    (Some(2), CmpOp::Eq) => Reason::AlwaysFalse,
+                    (Some(3), CmpOp::Eq) => Reason::AlwaysTrue,
```

Comment:
`== 3` is maybe a little dubious, but we were already doing it. (`==` is the _only_ case we handled before.)